ontology/yaml/resources/HVAC/entity_types/UH.yaml

```diff
@@ -35,3 +35,10 @@ UH_SS_DSP_HTZC:
   - SS
   - DSP
   - HTZC
+
+UH_FA:
```

Comment:
See above comment on `FA`.  Is `failed_alarm` truly the only point? What does the alarm logic consist of?

Reply:
Thanks for the comments. I will try to find out about alarm logic and get back to you.